tests/system/providers/google/cloud/tasks/example_tasks.py

```diff
@@ -48,22 +45,20 @@
 ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID")
 DAG_ID = "cloud_tasks_tasks"
 
-timestamp = timestamp_pb2.Timestamp()
-timestamp.FromDatetime(datetime.now() + timedelta(hours=12))
+TIMESTAMP = timestamp_pb2.Timestamp()
```

Comment:
May I know why do we want to make it upper case?

Reply:
restored to previous version